### PR TITLE
[1LP][RFR] get rid of CloudInfraProvider class

### PR DESCRIPTION
**Pull request by izapolsk**
1. CloudInfraProvider class is removed.
2. All its methods are added to both Cloud and Infra providers.
3. All CloudInfraProvider calls are replaced with CloudProvider and InfraProvider.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f2c46357a065e034715325bc543090c474744d20:

`cfme/infrastructure/provider/__init__.py`:
- :bangbang: Line 46:101: **E501** *line too long (102 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5568d79442f8d6075f52d2778e3300547c82dedf:

`cfme/infrastructure/provider/__init__.py`:
- :bangbang: Line 46:101: **E501** *line too long (102 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/common/cloudinfra_provider.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+from .provider import base_types
+
+# there are a lot of tests which request any of Cloud or Infra providers
+# CloudInfraProvider has been left in order not to
+# replace all such requests with both base Cloud and Infra providers
+CloudInfraProvider = [val for t, val in base_types().items() if t in ('infra', 'cloud')]  # noqa
```
Comment (RonnyPfannschmidt) [inline cfme/common/cloudinfra_provider.py]: cant this just be the mixin as aslias?

Comment (izapolsk) [reply cfme/common/cloudinfra_provider.py]: I just replaced all CloudInfra calls with CloudProvider and InfraProvider.

**Comment by dajoRH** [comment]
Lint report for commit 2335c74f78e723fa85911a505d94289910cf094f:

`cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py`:
- :bangbang: Line 22:101: **E501** *line too long (104 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**